basic/src/test/java/org/jboss/hal/testsuite/test/configuration/security/ACLTestCase.java

```diff
@@ -3,62 +3,62 @@
 import org.apache.commons.lang.RandomStringUtils;
 import org.jboss.arquillian.junit.Arquillian;
 import org.jboss.hal.testsuite.category.Shared;
-import org.jboss.hal.testsuite.dmr.AddressTemplate;
-import org.jboss.hal.testsuite.dmr.Operation;
-import org.jboss.hal.testsuite.dmr.ResourceAddress;
+import org.jboss.hal.testsuite.creaper.ResourceVerifier;
+import org.junit.AfterClass;
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Values;
 
 import java.io.IOException;
+import java.util.concurrent.TimeoutException;
 
-/**
- * @author Jan Kasik <jkasik@redhat.com>
- *         Created on 16.10.15.
- */
 @RunWith(Arquillian.class)
 @Category(Shared.class)
 public class ACLTestCase extends SecurityTestCaseAbstract {
 
-    private static final AddressTemplate ACL_TEMPLATE = SECURITY_DOMAIN_TEMPLATE.append("acl=classic/acl-module=*");
-    private static ResourceAddress ACL_ADDRESS;
-    private static String aclModule;
+    private static final Address ACL_ADDRESS = JBOSS_WEB_ADDRESS.and("acl","classic");
```

Comment:
Insert space before `"classic"` to comply with checkstyle rules and run at least `mvn clean test -Pbasic,rbac -Djboss.dist -DskipTests` next time.